README.md

```diff
@@ -9,20 +9,14 @@ To add a DNS record (e.g. a subdomain):
 4. save and quit
 5. Commit/push and wait for Docker hub to update
 6. Deploy to server2 according to [cloyne/servers](http://github.com/cloyne/servers)
-7. On server2, if it doesn't start properly (i.e., `pdns_server` does not show up in
-   `top` within the container), exec into the docker container
-   (`docker exec -ti dns script -q -c "/bin/bash" /dev/null`)
-8. edit `/etc/powerdns/pdns.conf` (you'll probably need to install an editor,
-   either `apt install nano` or `apt install vim-tiny`) and change the line
-   `launch=` to `launch=bind`. Then add the line
-   `bind-config=/etc/powerdns/bindbackend.conf`.
-9. Restart `pdns` with `sv restart powerdns`
-10. Run `pdns_control reload`. Now `pdns_control list-zones` should show
+7. To debug, on server2, exec into the docker container
+   (`docker exec -ti dns script -q -c "/bin/bash" /dev/null`) and
```

Comment:
BTW, I generally do `docker exec -t -i --entrypoint /bin/bash dns` to exec into the container with runit.